views/dashboard/components/homework_tiles.hbs

```diff
@@ -8,7 +8,7 @@
 	{{/if}}
 	<div data-testid="courses">
 		{{#each content}}
-			<a class="homework-link" href="{{this.url}}{{#if ../redirectToTab}}#activetabid={{../redirectToTab}}{{/if}}">
+			<a class="homework-link" aria-label="{{this.courseId.name}} - {{this.title}}" href="{{this.url}}{{#if ../redirectToTab}}#activetabid={{../redirectToTab}}{{/if}}">
```

Comment:
I'm not 100% sure if it applies here but this could be the case where the user should be informed about the redirect or opening a new tab: https://www.w3.org/TR/WCAG20-TECHS/G201.html

Reply:
When tabbing voiceover says it's on a link and you have to press enter to click it, some voiceovers have an info about redirect and opening a new tab. Why should we add more info in this particular case?

Reply:
This is due to the fact that not all screen readers might announce that it will open in the new window. 
I think this is a nice example of this warning:
![image](https://user-images.githubusercontent.com/25037067/88897566-c81dd580-d24b-11ea-9e1d-f23cdf4bc492.png)